docker-compose up fails with fresh git cloned dkron

**Describe the bug**
The `docker-compose up` command fails with following error with fresh clone of the repo.

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone https://github.com/distribworks/dkron.git`
2. `cd dkron`
3. `docker-compose up`
4. See error

**Error**
```
Step 8/11 : RUN go mod download
 ---> Using cache
 ---> 62661e7db007
Step 9/11 : COPY . .
 ---> Using cache
 ---> ed56f3bbbcc8
Step 10/11 : RUN go install ./...
 ---> Running in d00836574db6
dkron/ui.go:4:2: package embed is not in GOROOT (/usr/local/go/src/embed)
dkron/ui.go:7:2: package io/fs is not in GOROOT (/usr/local/go/src/io/fs)
ERROR: Service 'dkron' failed to build : The command '/bin/sh -c go install ./...' returned a non-zero code: 1
```

**Specifications**
 - OS: MacOS Big Sur 11.1

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.15
+FROM golang:1.16
 LABEL maintainer="Victor Castell <victor@victorcastell.com>"
 
 EXPOSE 8080 8946

